_studio/shared/include/mfx_utils.h

```diff
@@ -174,4 +175,9 @@ inline mfxStatus CheckAndDestroyVAbuffer(VADisplay display, VABufferID & buffer_
 }
 #endif
 
+bool operator==(mfxPluginUID const& l, mfxPluginUID const& r)
```

Comment:
this fails to build. You need `inline` I guess (or static inline).

Reply:
yes, looks like so